website/source/api/nodes.html.md

```diff
@@ -259,35 +262,110 @@ $ curl \
       "Details": null,
       "Message": "Node registered",
       "Subsystem": "Cluster",
-      "Timestamp": "2018-04-10T23:43:17Z"
+      "Timestamp": "2019-08-26T12:22:50+02:00"
     }
   ],
-  "HTTPAddr": "10.138.0.5:4646",
-  "ID": "f7476465-4d6e-c0de-26d0-e383c49be941",
+  "HTTPAddr": "127.0.0.1:4646",
+  "HostVolumes": {
```

Comment:
this won't be valid until 0.10?

Reply:
Correct - `master` is currently the `0.10` dev branch tho :)